plugins/Feedback/vue/src/RateFeature/RateFeature.vue

```diff
@@ -257,7 +257,7 @@ export default defineComponent({
       AjaxHelper.fetch({
         method: 'Feedback.sendFeedbackForFeature',
         featureName: this.title,
-        like: this.like ? '1' : '0',
```

Comment:
Suggest keeping this line as is otherwise both `'true'` and `'false'` may be true in php